src/components/sidebar/index.tsx

```diff
@@ -100,20 +107,37 @@ const Sidebar = ({
 	}
 
 	let sidebarContent
+	const filteredMenuItems = getFilteredNavItems(itemsWithMetadata, filterValue)
+	const navResourceItems = generateResourcesNavItems(currentProduct?.slug).map(
+		(item) => item
+	)
 	if (children) {
 		sidebarContent = children
-	} else {
-		const filteredMenuItems = getFilteredNavItems(
-			itemsWithMetadata,
-			filterValue
+	} else if (isInstallPage) {
+		sidebarContent = (
+			<OpenApiSidebarContents
+				navItems={filteredMenuItems as unknown as OpenApiNavItem[]}
+				navResourceItems={navResourceItems.splice(1)}
+				showFilterInput={false}
+			/>
```

Comment:
this allows for the highlighting of nav items on scroll like on the [api docs page](https://developer.hashicorp.com/hcp/api-docs/consul)